core/src/components/button/button.common.scss

```
@@ -306,3 +307,29 @@ ion-ripple-effect {
   background: #{var(--ion-toolbar-color, var(--background))};
   color: #{var(--ion-toolbar-background, var(--color))};
 }
+
+// Button Badge
+// --------------------------------------------------
+
+.button-native {
+  overflow: visible;
+
+  ion-ripple-effect {
+    // stylelint-disable-next-line property-disallowed-list
+    border-radius: inherit;
+  }
+
+  &::after {
```

This can be re-written for easier readability:

```suggestion
.button-native  ion-ripple-effect,
.button-native::after {
```